src/relay/backend/contrib/uma/targets.cc

```diff
@@ -62,16 +61,23 @@ TVM_REGISTER_GLOBAL("relay.backend.contrib.uma.RegisterTarget")
                                         relay::contrib::uma::RelayToTIR(target_name))
               .set_attr<FTVMTIRToRuntime>("TIRToRuntime", relay::contrib::uma::TIRToRuntime);
 
+      // target kind attrs inventory
+      auto kind = ::tvm::TargetKind::Get(target_name).value();
```

Comment:
Nit: aren't we already in the tvm namespace?

Reply:
Removed all such occurrence.